src/cljs/main/broadfcui/page/method_repo/create_method.cljs

```diff
@@ -35,6 +37,25 @@
     (assoc (utils/restructure namespace name) :snapshot-id snapshotId)))
 
 
+(defn- strip-comment-leaders
+  "Takes the comment (or blank) lines retrieved from the top of a WDL file (as a seq of strings),
+   finds the most common number of '#' characters to start non-blank lines, and removes that many
+   '#' characters from each line"
+  [comment-or-blank-lines]
+  (let [most-common-hash-count (->> comment-or-blank-lines
+                                    (remove string/blank?)
+                                    (map (comp count (partial re-find #"#+")))
+                                    frequencies (sort-by val) last key)
+        pattern (re-pattern (str "^#{" most-common-hash-count "}.*"))
+        trimmed-lines (map (fn [line]
+                             (string/trim
```

Comment:
Could you just use regex?
```cljs
(fn [line]
  (string/replace-first line #"^\#+\s+" ""))
```
(This is moot if we're explicitly keeping odd hashes, I just figure it might be less work)